### ISSUE TITLE
feature request: `-U` to unbury entire last command

### ISSUE THREAD

**Issue by ibarland** [MilesCranmer/rip2#70]
When using `rip` with wildcards, it'd be nice to have a way to unbury the entire last invocation of `rip`.

(I realize this'd mean an extra field "invocation-id" kept in `.record`, since even the millisecond timestamp may not work when somebody is calling `rip` from a script.   …though I guess just a single sentinel-character at the start of lines that were part of the most recent invocation would be fine too, if you're willing to frequently re-over-write the tail of the file.)

—— 
Btw, thanks for the awesome product!  I'd previously been using my own dumb little shell-script to get this affect, but it suffered from multiple flaws, one of which in particular kept annoying me.  I heard about `rip`/`rip2` on a [reddit thread a couple days ago](https://old.reddit.com/r/MacOS/comments/1he289e/macos_setup_for_devs_and_power_users/m20nys7/).

**Comment by MilesCranmer** [comment]
Good idea. I think the implementation sounds right too. Not sure the right long version though: `--unbury-all` seems like it would suggest unburying the entire graveyard.

This would be a breaking change due to the extra field; maybe we can make a function to automatically update the `.record` if needed (just assuming all lines have unique `invocation-id` or something). But probably a bit complex to start tracking that code around.
